### ISSUE TITLE
[cracklord-queued] Can't view Webpage. || Says Port 443 already in use.

### ISSUE THREAD

**Issue by Trig0n** [jmmcatee/cracklord#101]
<Backstory>
Fresh Ubuntu 14.04 Server.
Installed cracklord via deb.
Did not change the config.

<Problem>
Everytime I run cracklord with the standard config file, it prints as third line: Port 443 already in use.
But when I check netstat -tulpn, cracklord is the only process running on that port.
Then I kill the Process ID and try to restart it, same problem again.

Also when I try to visit cracklord after I started it (Going to xxx.xxx.xxx.xxx:443) it just tries to download a file and I don't see any bit of an interface.

Did I do something wrong? (Config is unchanged)

(Sorry for bad english/grammer, it's not my native language)

**Comment by emperorcow** [comment]
Two quick questions.  First, just confirming that CrackLord is started when you try and go to host:443?  And Second, what is the name of the file that it tries to download?  

My gut reaction is that you aren't doing anything wrong, there is either something else running on that port that is being sneaky / annoying, or we've got a bug!

P.S. - Your english was great!

**Comment by Trig0n** [comment]
Thanks for the fast response.

ps -A says cracklord-queued is running

The file it tries to download - a weird thing.
(On chrome on my phone):
It just shows it wants to download something, but it doesn't really do. It just shows 'unnamed file' in the statusbar.
It may be my phone, but I don't think so.

In before there was running tor on this port, but i edited the tor config and moved it to some other port, restarted tor, and checked netstat.
Nothing was running on Port 443.
Also I just killed tcpdump, which was also listening on port 443, but still the same error.

I just restarted my Server to make sure there aren't any processes running in screen or tmux anymore.

Still the same problem.
Apparently cracklord boots with the server? 
And is persistent?
Since I'm trying to kill it but a few seconds later it's running again with another processid.

I'm going to try it on my raspberry this evening and report if there are any errors.

**Comment by emperorcow** [comment]
Ok, a couple quick things.  First, you can stop the queue daemon from running on boot by overriding the upstart scripts with the following.  I think this should work, but if not please let me know:  

`echo manual | sudo tee /etc/init/cracklord-queued.override`

Second, once you have it back up, can you use wget on your local linux box and and go to https://localhost and see what it's trying to do?  Wget will give you several pieces of output on the command line. 

Third, in your cracklord-queued.conf what do you have as the LogLevel?  By default it's set to "Info", but if you change it to "Debug" you will get significantly more information which may also help us figure this out.

Also, just wanted to let you know that I'll be stepping away from my computer for most of the rest of the day.  It's a holiday here in the US and my family is visiting.  Any information you can get us about what it's trying to download, any oddities from the logs at debug, etc. would be helpful.

**Comment by Trig0n** [comment]
Well.
This is kinda awkward now.
What I did:
Stopped being retarded.

Since I thought the queue-server is like a flask-server, where I need to specify the :[port] in the browser, no matter what port i choose, I never thought about just using https://.

I just tried, as you told me in the first part of your response, https://[server ip] and it worked.
(Still kind of weird that it tries to download something if I access the port directly.)

I'm really sorry for the trouble and inconvenience I caused.

Happy holidays!
